server/lib/chat/chat-service.ts

```diff
@@ -360,6 +367,20 @@ export default class ChatService {
           return assertUnreachable(m.data)
       }
     })
+
+    let mentionIds = Set<SbUserId>()
```

Comment:
would probably prefer to use a native Set here, the immutable one really provides no benefits here